src/pages/public/imgae-gallery/Gallery.js

```diff
@@ -1,32 +1,45 @@
-import React, { useEffect, useState } from 'react'
-import { Grid } from '@mui/material';
-import AppLoader from '@components/Loader/AppLoader';
+import React, { useEffect, useState } from "react";
+import { Grid } from "@mui/material";
+import AppLoader from "@components/Loader/AppLoader";
 function Gallery() {
-    const [Gallery, setGallery] = useState(null)
-    useEffect(() => {
-        const urlNew = 'https://picsum.photos/v2/list';
-        fetch(urlNew).then(resp => resp.json()).then(resp => setGallery(resp))
-    }, [])
+  const [Gallery, setGallery] = useState(null);
+  useEffect(() => {
+    const urlNew = "https://picsum.photos/v2/list";
+    fetch(urlNew)
```

Comment:
haven't you used NetworkManager call here?

Reply:
I will use NetworkManager ill create a new branch for it